itis.playground/Pages/Game.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,301 @@
+/*
+ Правила
+ В игре присутствует 4 класса
+ Каждый бой состоит из нескольких раундов, игра заканчивается, когда все игроки одной из команд погибают
+ В каждом раунде сражается случайный игрок из первой команды - с случайным игроком из второй команды
+ Команды - это две случайные фракции по 4 игрока в каждой
+ Сражение происходит на землях первой команды, у случайного игрока из этой команды может быть преимущество локации - шанс 50%(один раз за игру)
+ Сила удара у каждого игрока случайна
+ У каждого игрока есть суперсила(зависит от класса) - можно использовать 1 раз за раунд(шанс 50%)
+ У каждого класса существует различная выносливость(после определенного уровня здоровья сила удара игрока снижается на 30%)
+ */
+
+import UIKit
+
+protocol Student {
+    var nameOfFraction: String { get }
+    var name: String { get }
```

Comment:
```suggestion
    let name: String { get }
```
не меняется то сделать константой